WindSpigot-Server/src/main/java/ga/windpvp/windspigot/knockback/KnockbackConfig.java

```diff
@@ -68,11 +68,25 @@ public static void init(File configFile) {
 			windpvpProfile.save(true);
 		}
 		
-		if (!keys.contains("vanilla")) {
-			final KnockbackProfile defaultProfile = new CraftKnockbackProfile("vanilla"); 
-			// Generated profiles are based on kohi kb values, not vanilla, so we set the values here
-			defaultProfile.setFrictionHorizontal(1.0);
-			defaultProfile.setFrictionVertical(1.0);
+		if (!keys.contains("kohi")) {
+			final KnockbackProfile defaultProfile = new CraftKnockbackProfile("kohi"); 
+
+			defaultProfile.setHorizontal(0.35);
+                        defaultProfile.setRodHorizontal(0.35);
+                        defaultProfile.setArrowHorizontal(0.35);
+                        defaultProfile.setPearlHorizontal(0.35);
+                        defaultProfile.setSnowballHorizontal(0.35);
+                        defaultProfile.setEggHorizontal(0.35);
+                        defaultProfile.setAddHotizontal(0.425);
+
+			defaultProfile.setVertical(0.35);
+                        defaultProfile.setRodVertical(0.35);
+                        defaultProfile.setArrowVertical(0.35);
+                        defaultProfile.setPearlVertical(0.35);
+                        defaultProfile.setSnowballVertical(0.35);
+                        defaultProfile.setEggVertical(0.35);
+                        defaultProfile.setAddVertical(0.085);
```

Comment:
```suggestion
                        defaultProfile.setExtraVertical(0.425);
```
Add vertical is actually added to knockback, extra vertical is the additional knockback multiplier for player entities.